bdk-ffi-bindgen/Cargo.toml

```diff
@@ -6,5 +6,5 @@ edition = "2021"
 [dependencies]
 anyhow = "=1.0.45" # remove after upgrading to next version of uniffi
 structopt = "0.3"
-uniffi_bindgen = "0.19.5"
+uniffi_bindgen = "=0.20.0"
```

Comment:
Please remove pinning here, or just rebase this one after #203 is fixed and merged.

Reply:
I'll rebase once #203 is merged. I fixed it there, so if you can approve it we'll merge right away and rebase this one.